docs/user/openstack/install_upi.md

```diff
@@ -314,22 +302,25 @@ We will create three files:
 
 **`/etc/hostname`**:
 
-```
+```sh
+$ cat /etc/hostname
```

Comment:
This is extraneous. Please remove it. This should just show the file contents to make it easy to copy/paste.